### DISCUSSION TITLE
Ethernet RJ45 port

### DISCUSSION THREAD

**Discussion by Ian-Johnston** [revk/xESP32-Faikout#170] [General]
Is it possible to give my Daikin aircon an RJ45 port (ethernet). I am running WiFi using the original Daikin WiFi module via my own VB software........but I'd rather have a cabled connection. Possible?
Ian.

**Comment by revk** [comment]
Interesting idea - the ESP32 I am using can, in theory, be wired for Ethernet.

Indeed, I think there are ESP32 dev boards that exist with Ethernet and even PoE.

Loading my Faikin code on to such a board may be possible. But my code is not designed to work with the internal Ethernet controller, so won't just work, sadly. So it will probably have to wait until I have done stuff with the embedded Ethernet controller.

**Comment by Ian-Johnston** [comment]
Thanks, will be following here just incase you implement this.

**Reply by revk** [reply]
Wired ethernet is on my list, but no idea when, thanks.

**Comment by Ian-Johnston** [comment]
My VB app runs on my local webserver, mainly to control my workshop temps......but also picks up some other data/sensors.

![image](https://github.com/revk/ESP32-Faikin/assets/61011024/9793b66c-e7a2-49b1-b8e1-b7a912bbc05f)
